easybuild/scripts/bootstrap_eb.py

```diff
@@ -448,7 +459,9 @@ def main():
     for path in orig_sys_path:
         include_path = True
         # exclude path if it's potentially an EasyBuild package
-        if 'easybuild' in path:
+        if 'easybuild' in path or os.path.exists(os.path.join(path, 'easybuild')):
+            include_path = False
+        if '/vsc_' in path or os.path.exists(os.path.join(path, 'vsc')):
```

Comment:
well, good luck we didn't mount our filesystems under `/vsc_mnt` then..... (my point is, this is just excluding some random strings, have nothing to do with python modules)